service/src/main/java/com/plhal/ares/service/DataServiceImpl.java

```diff
@@ -27,4 +31,7 @@ public Firma najdiFirmu(String ico) {
         return dataRepository.najdiFirmu(ico);
     }
 
+    public void pridejFirmu(Firma firma) {
```

Comment:
prosim u novych veci uz anglicke nazvy.

Reply:
ZKuste u novych veci pridavat uz i ten javadoc. Donuti to cloveka premyslet i nad implementaci.

Reply:
A pouzivejte prosim i lombok anotace. Zejmena @NonNull